### ISSUE TITLE
Application not reacting to input

### ISSUE THREAD

**Issue by DanChaltiel** [cecilesauder/bibliographeR#1]
Hi Cecile,

After your awesome presentation at the UseR2019, I really wanted to try your app.

Unfortunately, weather I use `bibliographeR::run_app()` in RStudio or the [web link ](https://cecilesauder.shinyapps.io/bibliographer/) you provided, the app is not reacting to my input.

After entering a keyword, is there any key I should press to get a result?

**Comment by Felixmil** [comment]
Same here, I wanted to try it out as soon as It was presented at user2019 (great presentation btw!). 
I saw that there's no reactive part in the `server` side of the app: https://github.com/cecilesauder/bibliographeR/blob/399dba89a67ff403f063c085a29ff194b6c4432d/inst/BibliographeR/app.R#L47-L50

**Comment by cecilesauder** [comment]
Yes, sorry for that, as I said at useR it's a work in progress... and the app is not available for now but I'm working on it. As you see the server part is totally empty for the moment :-/

**Comment by DanChaltiel** [comment]
No problem this is perfectly understandable :-)
(But maybe you should state this in some readme file)

And regardless of the shiny app, wouldn't there be some piece of code we could adapt with our keywords?
